assets/agenda/components/AgendaListItem.tsx

```diff
@@ -132,7 +132,9 @@ class AgendaListItem extends React.Component<IProps> {
                 'wire-articles__item--not-covering': !hasCoverages(this.props.item),
                 'wire-articles__item--postponed': isPostponed(this.props.item),
                 'wire-articles__item--canceled': isCanceled(this.props.item),
-                'wire-articles__item--visited': this.props.isRead,
+
+                // Planning items don't have a version, while event items do, which is affecting the `isRead` prop.
```

Comment:
This looks like a wrong place to do it. This line looks good
`'wire-articles__item--visited': this.props.isRead,`

if item is read - apply visited style. If `isRead` is not indicating correctly, it should be fixed in a parent component rather than adjusting for incorrectness here.

Reply:
Yes, that is the second option. I went with this one, because I can't know all the item states, and didn't want to risk breaking anything.